lms/services/digest.py

```diff
@@ -81,7 +81,7 @@ class UnifiedCourse:
 
     authority_provided_id: str
     title: Optional[str]
-    instructors: Tuple[UnifiedUser]
+    instructor_h_userids: Tuple[str]
```

Comment:
There's no functional differences (and you argue for the immutability of tuples here) but I find a bit counter-intuitive to have a collection of the same thing using tuples

https://stackoverflow.com/a/16941245